packages/chain-adapters/src/evm/avalanche/AvalancheChainAdapter.ts

```diff
@@ -94,15 +91,15 @@ export class ChainAdapter extends EvmBaseAdapter<KnownChainIds.AvalancheMainnet>
 
     return {
       fast: {
-        txFee: getTxFee({ gasLimit, ...fast }),
+        txFee: bnOrZero(bn(fast.gasPrice).times(gasLimit)).toFixed(0),
```

Comment:
since `gasPrice` and `maxFeePerGas` are the same value now, we can simplify the `txFee` calculation